recipes/file-access-mode-constants/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@nodejs/file-access-mode-constants",
+  "version": "1.0.0",
+  "description": "Handle DEP0176 via transforming imports of `fs.F_OK`, `fs.R_OK`, `fs.W_OK`, `fs.X_OK` from the root `fs` module to `fs.constants`.",
+  "type": "module",
+  "scripts": {
+    "test": "npx codemod@next jssg test -l typescript ./src/workflow.ts ./"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nodejs/userland-migrations.git",
+    "directory": "recipes/file-access-mode-constants",
+    "bugs": "https://github.com/nodejs/userland-migrations/issues"
+  },
+  "author": "nekojanai (Jana)",
+  "license": "MIT",
+  "homepage": "https://github.com/nodejs/userland-migrations/blob/main/recipes/file-access-mode-constants/README.md",
+  "dependencies": {
+    "@ast-grep/napi": "^0.39.2",
+    "@nodejs/codemod-utils": "^0.0.0"
+  },
```

```suggestion
  "dependencies": {
    "@nodejs/codemod-utils": "*"
  },
```

`*` mean monorepo one